opencti-platform/opencti-front/src/components/InputSliderField.tsx

```diff
@@ -0,0 +1,118 @@
+import { Field, FieldProps } from 'formik';
+import React, { FunctionComponent } from 'react';
+import { Slider } from '@mui/material';
+import TextField from './TextField';
+import { SubscriptionFocus } from './Subscription';
+import { useLevel } from '../utils/hooks/useScale';
+
+interface InputSliderFieldProps {
+  label: string;
+  variant?: string;
+  onSubmit?: (name: string, value: string | number | number[]) => void;
+  onFocus?: (name: string, value: string) => void;
+  editContext?: readonly ({
+    readonly focusOn: string | null;
+    readonly name: string;
+  } | null)[] | null
+  containerStyle?: Record<string, string | number>;
+  entityType: string;
+  attributeName: string;
+  disabled?: boolean;
+}
+
+const InputSliderField: FunctionComponent<InputSliderFieldProps & FieldProps> = ({
+  form: { setFieldValue },
+  field: { name, value },
+  label,
+  variant,
+  onFocus,
+  onSubmit,
+  containerStyle,
+  editContext,
+  entityType,
+  attributeName,
+  disabled,
+}) => {
+  const { level: { color }, marks } = useLevel(entityType, attributeName, value);
+
+  const min = marks.length > 0 ? marks[0].value : 0;
+  const max = marks.length > 0 ? marks[marks.length - 1].value : 0;
+
+  const sliderMarks = marks.map((mark) => ({
```

Comment:
maybe we don't need anymore this block and can directly use `marks` instead of `sliderMarks` (since we removed the label translation) @SarahBocognano